frontend/src/scenes/funnels/funnelDataLogic.test.ts

```diff
@@ -164,14 +164,9 @@ describe('funnelDataLogic', () => {
         })
     })
 
-    /**
-     * We set insightLogic.insight via a call to setInsight for data exploration,
-     * in future we should use the response of dataNodeLogic via a connected
-     * insightDataLogic.
-     */
-    describe('based on insightLogic', () => {
+    describe('based on insightDataLogic', () => {
         describe('results', () => {
-            it('with non-funnel insight', async () => {
+            it.skip('with non-funnel insight', async () => {
```

Comment:
skip 🙈

Reply:
The `.skip` is actually intentional here, as I've removed the condition that test if we currently have a funnel result (versus the result of another insight) and plan to re-add that with the data manager.